src/View/Antlers/Language/Runtime/NodeProcessor.php

```diff
@@ -1538,10 +1542,20 @@ public function reduce($processNodes)
 
                             $sectionName = 'section:'.$tagMethod.'__yield'.$activeYield;
 
+                            $sections = null;
+
+                            if ($this->cascade == null && $sections == null) {
```

Comment:
If you're setting sections to null on the line before, you don't need it in this condition.